Hunting Queries/Microsoft 365 Defender/Command and Control/SuspiciousNetworkConnection-Googleapis.yaml

```diff
@@ -0,0 +1,18 @@
+id: fe4f4336-a2e7-4cae-af43-3210106475be
+name: Suspicious network connection to Google APIs - Possible GC2-Sheet C2
+description: |
+  This query will look for connections made to several Google cloud service API endpoints which are not associated with a legitimate browser or application. The query was inspired by the GC2-Sheet project: https://github.com/looCiprian/GC2-sheet. GC2-Sheet is a C2 platform designed to leverage Google Sheets and Google Drive SaaS applications.
+requiredDataConnectors:
+- connectorId: MicrosoftThreatProtection
```

Comment:
YAML is not valid. There should be additional tab in these three lines.
Example: https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/Azure%20Activity/Hunting%20Queries/AnalyticsRulesAdministrativeOperations.yaml

Reply:
Apologies for the delay. I had some personal things to attend to. I have amended the YAML and checked in VSCode. I have also added extend statements and field mappings as requested. I'll just check everything is looking good after the initial tests.